srtcore/core.cpp

```diff
@@ -228,6 +228,7 @@ CUDT::CUDT(CUDTSocket* parent): m_parent(parent)
     m_iIpV6Only             = -1;
     // Runtime
     m_bRcvNakReport             = true; // Receiver's Periodic NAK Reports
+    m_PacingMode                = SRT_PACING_UNSET; // API/ABI backward compatible mode
```

Comment:
It make more sense if you simply add kinda `m_bHybridPacing` boolean field. This is only required to distinguish the INPUTBW and HYBRIDBW modes. Then, you just set:

* UNLIMITED (default): MAXBW = -1, others ignored
* MAXBW: MAXBW as provided, others ignored
* INPUTBW: MAXBW = 0, INPUTBW as provided, OHEADBW as provided, HYBRIDPACING = false
* ESTINPUTBW: MAXBW = 0, INPUTBW = 0, OHEADBW as provided, HYBRIDPACING = false
* HYBRIDBW: MAXBW = 0, INPUTBW as provided, OHEADBW as provided, HYBRIDPACING = true

Decoding the mode from fields:
* If MAXBW == -1, report UNLIMITED
* If MAXBW > 0, report MAXBW
* if MAXBW == 0:
   * if INPUTBW == 0, report ESTINPUTBW
   * if INPUTBW > 0:
      * if HYBRIDPACING is false, report INPUTBW
      * otherwise report HYBRIDBW